index.bs

```diff
@@ -297,6 +418,7 @@ dictionary SanitizerConfig {
 };
 </pre>
 
+ISSUE: Sould members be required, or have declared defaults?
```

Comment:
(Same as above, pls file ISSUE for those that area only in the document)

Reply:
Done. (Also removed issues that had been resolved, like this one, or otherMarkup.